src/Service/AuthorizationService.php

```diff
@@ -117,33 +117,33 @@ public function isGranted($identity, $permission, $context = null)
         }
 
         if ($this->hasAssertion($permission)) {
-            return $this->assert($this->assertions[(string) $permission], $identity, $context);
+            return $this->assert($this->assertions[(string) $permission], $permission, $identity, $context);
```

Comment:
I think we should keep that.

$permission is not $this->assertions[(string) $permission]

And permission is always a string, is is not?

> Op 5 sep. 2016 om 20:18 heeft Michaël Gallego <notifications@github.com> het volgende geschreven:
> 
> In src/Service/AuthorizationService.php:
> 
> > @@ -117,33 +117,33 @@ public function isGranted($identity, $permission, $context = null)
> >          }
> >  
> >          if ($this->hasAssertion($permission)) {
> > -            return $this->assert($this->assertions[(string) $permission], $identity, $context);
> > +            return $this->assert($this->assertions[(string) $permission], $permission, $identity, $context);
> Actually we have this interface (https://github.com/ZF-Commons/zfc-rbac/blob/develop/src/Assertion/AssertionInterface.php) and it does not implement __toString.
> 
> Maybe we should remove this interface completely?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>